apps/election-manager/src/__snapshots__/app.test.tsx.snap

```diff
@@ -144,7 +144,7 @@ exports[`printing ballots, print report, and test decks 1`] = `
       >
         Election Hash: 
         <strong>
-          6bdcd6d885
+          61cf15c77e
```

Comment:
why did this change?

Reply:
Because of this: https://github.com/votingworks/vxsuite/pull/1188/files#diff-3bbfed999ed23f50545ca95915cabc176aed8a7cef389532d0ecf6b7105da868R34-R36.